src/core/variable.jl

```diff
@@ -888,9 +803,13 @@ function variable_mc_generation_active(pm::_PMs.AbstractPowerModel; nw::Int=pm.c
     )
 
     if bounded
-        for (i,gen) in _PMs.ref(pm, nw, :gen), c in cnds
-            JuMP.set_lower_bound(pg[i][c], gen["pmin"][c])
-            JuMP.set_upper_bound(pg[i][c], gen["pmax"][c])
+        for (i,gen) in _PMs.ref(pm, nw, :gen)
+            if haskey(gen, "pmin")
+                JuMP.set_lower_bound.(pg[i], gen["pmin"])
+            end
+            if haskey(gen, "pmax")
+                JuMP.set_upper_bound.(pg[i], gen["pmax"])
```

Comment:
should we instead be using the internal `set_lower_bound` and `set_upper_bound` here?